modules/sdk-coin-avaxp/src/avaxp.ts

```diff
@@ -79,15 +82,46 @@ export class AvaxP extends BaseCoin {
       case TransactionType.AddDelegator:
       case TransactionType.AddValidator:
         if (!params.txParams.recipients || params.txParams.recipients.length === 0) {
-          const filteredRecipients = [{ address: stakingOptions.nodeID, amount: stakingOptions.amount }];
+          const filteredRecipients = [{ address: stakingOptions?.nodeID, amount: stakingOptions?.amount }];
```

Comment:
staking options should not be `?` ? as it will always be there for `addValidator` (if i remember correctly )